js/cryptocom.js

```diff
@@ -921,13 +968,14 @@ module.exports = class cryptocom extends Exchange {
         }
         const request = {};
         const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchOrder', market, params);
-        if (marketType === 'spot') {
+        if ((marketType === 'spot') || (marketType === 'spot')) {
```

Comment:
Is this a bug?
It seems right hand side should be marketType == 'margin'.

Reply:
@kroitor

Reply:
#14107